jupyter_book/config.py

```diff
@@ -171,6 +161,18 @@ def get_final_config(
         paths_static.append("_static")
         sphinx_config["html_static_path"] = paths_static
 
+    if not use_external_toc:
```

Comment:
So the case that we *don't* use the external TOC is if users are directly putting `toctree` in their pages, right? If so we should document this.

Reply:
no this is for the case when you are using the `individualpages` option (se the `build` function)